packages/contracts-periphery/contracts/universal/op-nft/Optimist.sol

```diff
@@ -105,17 +119,15 @@ contract Optimist is ERC721BurnableUpgradeable, Semver {
     }
 
     /**
-     * @notice Checks whether a given address is allowed to mint the Optimist NFT yet. Since the
-     *         Optimist NFT will also be used as part of the Citizens House, mints are currently
-     *         restricted. Eventually anyone will be able to mint.
+     * @notice Checks OptimistAllowlist to determine whether a given address is allowed to mint
+     *         the Optimist NFT yet. Since the Optimist NFT will also be used as part of the
+     *         Citizens House, mints are currently restricted. Eventually anyone will be able
+     *         to mint.
```

Comment:
```suggestion
     *         the Optimist NFT. Since the Optimist NFT will also be used as part of the
     *         Citizens House, mints are currently restricted. Eventually anyone will be able
     *         to mint.
```
nit